### ISSUE TITLE
Umap computation time

### ISSUE THREAD

**Issue by jboelaert** [emilienschultz/activetigger#142]
Umap computation seems very slow. On 7000 obs * 600 features it takes 1'30-2'00, compared to around 10 seconds on R. 
Maybe there's a faster Python package for umap out there?

**Comment by emilienschultz** [comment]
Yes, there is a faster package, for the moment I haven't taken the time to
do so, I will

**Comment by emilienschultz** [comment]
I installed cuml.UMAP and it is fast now ! Only point : the install should be managed accordingly to the CUDO version, so it is a specific step to do in the install, in mention it in the doc
